private_gpt/ui/ui.py

```diff
@@ -409,11 +409,55 @@ def _build_ui_blocks(self) -> gr.Blocks:
                         inputs=system_prompt_input,
                     )
 
+                    def get_model_label() -> str:
+                        """Get model label from llm mode setting YAML.
+
+                        Raises:
+                            ValueError: If an invalid 'llm_mode' is encountered.
+
+                        Returns:
+                            str: The corresponding model label.
+                        """
+                        # Get model label from llm mode setting YAML
+                        # Labels: local, openai, openailike, sagemaker, mock, ollama
+                        config_settings = settings()
+                        if config_settings is None:
+                            raise ValueError("Settings are not configured.")
+
+                        # Get llm_mode from settings
+                        llm_mode = config_settings.llm.mode
+
+                        # Mapping of 'llm_mode' to corresponding model labels
+                        model_mapping = {
+                            "local": config_settings.local.llm_hf_model_file,
+                            "openai": config_settings.openai.model,
+                            "openailike": config_settings.openai.model,
+                            "sagemaker": config_settings.sagemaker.llm_endpoint_name,
+                            "mock": llm_mode,
+                            "ollama": config_settings.ollama.model,
+                        }
+
+                        try:
+                            return model_mapping[llm_mode]
+                        except KeyError:
+                            raise ValueError(
```

Comment:
Instead of raising an error, we should return None to not break the execution with new models added

Reply:
Updated to return None https://github.com/imartinez/privateGPT/pull/1648/commits/b12d7f8b630449b8641f3422b4efddb9f497a35c